.circleci/config.yml

```diff
@@ -89,14 +105,16 @@ jobs:
     executor: browser-node
     steps:
       - use_workspace
+      - browser-tools/install-chrome
+      - browser-tools/install-chromedriver
       - run:
           name: Install Playwright
           command: |
             npx playwright install
       - run:
           name: Install Fonts
           command: ./.circleci/install-fonts.sh
-      - run: yarn run test:rendering
+      - run: pnpm run test:rendering
       - store_test_results:
           path: ./test/rendering/reports/xml/report.xml
```

Comment:
This step seems to fail silently on ci. Might be the case on `main/master` as well.
> Unable to save test results from /home/circleci/dist-flow-charts/test/rendering/reports/xml/report.xml
Error error accessing path "/home/circleci/dist-flow-charts/test/rendering/reports/xml/report.xml": stat /home/circleci/dist-flow-charts/test/rendering/reports/xml/report.xml: no such file or directory
Found no test results, skipping